notebooks/test_notebooks.py

```diff
@@ -17,6 +18,9 @@ def run_notebook(path):
     proc = ExecutePreprocessor(timeout=600, kernel_name='python3')
     proc.allow_errors = True
 
+    if sys.version_info[0] == 3 and sys.version_info[1] >= 8 and sys.platform.startswith('win'):
+        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
Why does this `if` check for?

Reply:
https://github.com/jupyter/nbconvert/issues/1372

Reply:
Add a comment to explain the problem and link to the issue.